### DISCUSSION TITLE
nhibernate unit of work full source code

### DISCUSSION THREAD

**Discussion by junix2020** [nhibernate/nhibernate-core#2572] [General]
Anybody here still have the full source code of NHibernate Unit of Work example?

**Comment by gliljas** [comment]
https://nhibernate.info/doc/patternsandpractices/nhibernate-and-the-unit-of-work-pattern

**Reply by bahusoid** [reply]
Source code link from article is dead: http://hibernatingrhinos.googlecode.com/svn/trunk/UnitOfWork/

Still you can download all files via web archive: https://web.archive.org/web/20110908053146/http://hibernatingrhinos.googlecode.com/svn/trunk/UnitOfWork/src/
